master/static/srv/entrypoint.sh

```diff
@@ -3,6 +3,13 @@
 set -e
 set -x
 
+# Create symbolic links from well-known files to this process's STDOUT and
+# STDERR. Anything written to those files will be inserted into the output
+# streams of this process, allowing distributed training logs to route through
+# individual containers rather than all going through SSH back to agent 0.
+ln -sf /proc/$$/fd/1 /run/determined/train/determined.log
+ln -sf /proc/$$/fd/2 /run/determined/train/determined_error.log
```

Comment:
non-blocking: _Totally_ minor, but I think using the names `{stdout,stderr}.log` or so would be more nicely symmetric while also avoiding redundancy (we've already got `determined` and `train` in the path; the base names can be a bit more specific).